pkg/v1/tkg/test/tkgctl/tkgs_cc/tkgs_cc_workload_cluster_test.go

```diff
@@ -108,9 +108,6 @@ var _ = Describe("TKGS ClusterClass based workload cluster tests", func() {
 			deleteClusterOptions = getDeleteClustersOptions(e2eConfig)
 			clusterOptions.ClusterName = e2eConfig.WorkloadClusterOptions.ClusterName
 			clusterOptions.Namespace = e2eConfig.WorkloadClusterOptions.Namespace
-
-			// use a custom cluster class config file with custom ClusterBootstrap and Antrea resources to
-			// verify addons are successfully installed
```

Comment:
there is nothing particular to antrea in this section of code. Also, there is no addons verification here. The addons verification happens further and the It comment makes it pretty obvious, so removing this comment.